template.yaml

```diff
@@ -23,6 +23,10 @@ Globals:
       AllowHeaders: "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token, Access-Control-Allow-Origin'"
 
 Parameters:
+  CognitoAuthorizerArn:
+    Type: 'AWS::SSM::Parameter::Value<String>'
+    Description: Reference to Cognito UserPool for the stage
+    Default: CognitoAuthorizerArn
```

Comment:
Not in use complaints from linter

Reply:
linter doesn't understand the include, it is referenced in docs/cristin-proxy-swagger.yaml

Reply:
Can you add an lint exclude so we dont get an error?